src/authInfo.ts

```diff
@@ -77,7 +75,7 @@ export interface AuthFields {
   usernames?: string[];
   userProfileName?: string;
   expirationDate?: string;
-}
+};
 
 export type Authorization = {
```

Comment:
I wonder if we will need other kinds of Authorization.

```
export type Authorization = {
   loginUrl: string;
   token: string;
   timestamp: string;
}

export type OrgAuthorization = Authorization & {
   orgId: string // should this be on environment rather than authorization?
   authMethod: ...
}

export FunctionAuthorization = Authorization & {
 ....
}
```

Do we need all types of auth specified here or do plugins have this info and it is just stored through core because of the json map?